src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private double area;
+    private double perimeter;
+    private Color color;
+    
+    public abstract void draw();
+    
+    public abstract String getParameters();
```

Comment:
these methods better to move to the separate interfaces:
* interface - behavior
* abstract class - state